### ISSUE TITLE
Installation fails with KeyError

### ISSUE THREAD

**Issue by baudm** [baudm/mplayer.py#17]
Original [issue 14](https://code.google.com/p/python-mplayer/issues/detail?id=14) created by baudm on 2013-05-20T16:51:18.000Z:

<b>What steps will reproduce the problem?</b>
1. Download mplayer.py-0.7.0.tar.gz
2. Run &quot;sudo python setup.py install&quot;

<b>What is the expected output? What do you see instead?</b>
- Expectation: successful installation.
- Instead of successfully installing, a &quot;KeyError: u'UNKNOWN'&quot; is shown.

<b>What version of the product are you using? On what operating system?</b>
- mplayer.py v0.7.0 on Linux Mint 14 MATE edition, x64

<b>Please provide any additional information below.</b>
- Fix:

In core.py, line 252, change this:

t = mtypes.type_map[arg]
sig.append('{0}{1}{2}'.format(t.name, i, optional))
types.append('mtypes.{0},'.format(t.**name**))

to this:

if(arg in mtypes.type_map):
    t = mtypes.type_map[arg]
    sig.append('{0}{1}{2}'.format(t.name, i, optional))
    types.append('mtypes.{0},'.format(t.**name**))

This gets rid of the KeyError during installation.

**Comment by pmuller** [comment]
Same issue with mplayer.py 0.7.0 on a raspberry pi (raspbian) with package mplayer2 2.0-728-g2c378c7-4+b1 :
```$ pip install mplayer.py
Collecting mplayer.py
  Using cached mplayer.py-0.7.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-p1imi5zn/mplayer.py/setup.py", line 5, in <module>
        from mplayer import __version__
      File "/tmp/pip-build-p1imi5zn/mplayer.py/mplayer/__init__.py", line 37, in <module>
        from mplayer.core import Player, Step
      File "/tmp/pip-build-p1imi5zn/mplayer.py/mplayer/core.py", line 412, in <module>
        Player.introspect()
      File "/tmp/pip-build-p1imi5zn/mplayer.py/mplayer/core.py", line 308, in introspect
        cls._generate_methods()
      File "/tmp/pip-build-p1imi5zn/mplayer.py/mplayer/core.py", line 292, in _generate_methods
        func = cls._gen_method_func(name, args)
      File "/tmp/pip-build-p1imi5zn/mplayer.py/mplayer/core.py", line 252, in _gen_method_func
        t = mtypes.type_map[arg]
    KeyError: '2.0-728-g2c378c7-4+b1'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-p1imi5zn/mplayer.py/
```

No issue with with the master branch. Apparently you fixed it with commit 03fd9240a4515dc8165f515e005a10f1c179db32.

Any plan for a new release? :)

**Comment by baudm** [comment]
Already fixed. @pmuller v0.7.1 was released several months ago. Sorry I haven't been attending to the open issues lately.
